web/NetFrenzy.html

```diff
@@ -84,11 +84,21 @@
 			<div id="queryui">
 				<input name="query" value="" type="text" id="query">
 				<div id="query-buttons">
-					<input type="submit" title="run query" id="runquery" value="▶" onclick="runQuery()">
-					<input type="submit" value="↕" title="toggle multiline" id="linetoggle" onclick="toggleInputMode()">
-					<input type="submit" value="⏸" title="stabilize graph" onclick="window.viz.stabilize()">
-					<input type="submit" value="𝒾" title="show info" onclick="toggleInfo()">
-					<input type="submit" value="𝒽" title="show history" onclick="toggleHistory()">
+					<button title="run query" id="runquery" onclick="runQuery()">
+						<img src="icons/play-fill.svg">
```

Comment:
Please vertically align the icons in the buttons and ensure all icons are of the same size. For example, the play and pause icons are much smaller than the history icon
![image](https://user-images.githubusercontent.com/5117371/157477465-10e5475e-45ad-4ebd-a458-c51a1cd1c04b.png)